src/i18n/locales/mr_IN/translation.json

```diff
@@ -345,10 +345,35 @@
           "placeholder": "config=https://your-btc-pay.org/lnd-config/212121/lnd.config"
         },
         "errors": {
-          "connection_failed": "Connection failed. Is your Core Lightning node online and using the commando plugin?"
+          "connection_failed": "कनेक्शन अयशस्वी झाले. तुमचा कोर Lightning node ऑनलाइन आहे आणि कमांडो commando plugin आहे का?"
         }
       }
     },
+    "home": {
```

Comment:
Should this part be translated as well?

Reply:
No, actually because the `home` is working as a property in that JSON file so translating that property would require us to translate/replace it in the entire codebase.
What are your thoughts?

Reply:
Sorry, this is a misunderstanding. When I wrote this this section was still in English. Didn't mean the word `home`. All good. Thanks!